### ISSUE TITLE
Double-bullets for lists

### ISSUE THREAD

**Issue by jancborchardt** [stefan-niedermann/nextcloud-notes#332]
When you create a list with dashes, this happens:
![](https://i.imgur.com/uWhS7Lz.png)

Instead, the dashes should simply be used as list bullets, no additional dots shown.

**Comment by korelstar** [comment]
Why not tracking upstream issues like this one? It's definitely an issue in this app which should be fixed. It would be good to have an overview of problems with libraries we use in this app in order to be able to make reasonable decisions like updates or switching the library.

**Comment by stefan-niedermann** [comment]
Because people expect issues getting fixed if they stay here open, and this will not happen in this context. Most people open issues upstream, so we can take car about what's and what's not working in our app.

Tracking them here leads to duplicate discussion, wrong expections and a cluttered issue tracker with long running tasks that can never be resolved here.

**Comment by stefan-niedermann** [comment]
I don't think so. Ghey will close it, if it is a duicate, so simply submit an issue :)
